### ISSUE TITLE
256 bucket

### ISSUE THREAD

**Issue by TeslaCoil95** [madMAx43v3r/chia-plotter#408]
Does anyone have a problem when running on 256 buckets , plotting just stops?

**Comment by lordzamura** [comment]
no. all is ok.

**Comment by ataa** [comment]
256 and 512 buckets work fine, are you on Windows?

**Comment by lordzamura** [comment]
yes. windows is working good. dont forget. bucket 512 4 kat fazla ram kullanır. default is 4 core 2 gb ram.

**Comment by 69chargr** [comment]
> Does anyone have a problem when running on 256 buckets , plotting just stops?

I do. Freezes in the second line of phase 3 every time.

**Comment by konkersgg** [comment]
> yes. windows is working good. dont forget. bucket 512 4 kat fazla ram kullanır. default is 4 core 2 gb ram.

mine NEVER goes above 1.4GB RAM with 10 threads and 256 buckets lol. weird af, but it finishes much faster than original plotter. 1h35m vs 8h40m for a single plot.

**Comment by madMAx43v3r** [comment]
and with 128 buckets no issue?
